services/motion/motion.go

```diff
@@ -56,7 +56,7 @@ type MoveOnGlobeReq struct {
 	Obstacles []*spatialmath.GeoGeometry
 	// Optional motion configuration
 	MotionCfg *MotionConfiguration
-
+	// Set of obstacles which the robot must remain within while navigating
 	BoundingRegions []*spatialmath.GeoGeometry
```

Comment:
Can you move this to be with the obstacles?

Reply:
A bounding region is not an obstacle...

Reply:
@nicksanford definition taken from API 
https://github.com/viamrobotics/api/blob/main/proto/viam/service/motion/v1/motion.proto#L166

Reply:
Nick is right, we should remove the word "obstacle" when referring to these

Reply:
ok, what do we think of:
```
Set of bounds which the robot must remain within while navigating
```


------------------------------------------------------------------------------------------
Is it worth updating the API to match the new definition we are putting forward?